app/_src/guides/gateway-api.md

```diff
@@ -0,0 +1,336 @@
+
+You may not have support for `LoadBalancer` if you are running locally with `kind` or `k3d`.
+One option for `kind` is [kubernetes-sigs/cloud-provider-kind](https://github.com/kubernetes-sigs/cloud-provider-kind) may be helpful.
+{% endwarning %}
+
+You can now check if gateway is running in demo app `kuma-demo` namespace:
+```shell
+kubectl get pods -n kuma-demo
+```
+Observe the gateway pod:
+```shell
+NAME                       READY   STATUS    RESTARTS   AGE
+demo-app-d8d8bdb97-vhgc8   2/2     Running   0          5m
+kuma-cfcccf8c7-hlqz5       1/1     Running   0          20s
+redis-5484ddcc64-6gbbx     2/2     Running   0          5m
+```
+
+Retrieve the public URL for the gateway with:
+```shell
+export PROXY_IP=$(kubectl get svc --namespace kuma-demo kuma -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
+echo $PROXY_IP
+```
+
+Check the gateway is running:
+```shell
+curl -v ${PROXY_IP}:8080
+```
+
+Which outputs:
+```shell
+*   Trying 127.0.0.1:8080...
+* Connected to 35.226.116.24 (35.226.116.24) port 8080
+> GET / HTTP/1.1
+> Host: 127.0.0.1:8080
+> User-Agent: curl/8.7.1
+> Accept: */*
+>
+* Request completely sent off
+< HTTP/1.1 404 Not Found
+< content-length: 62
+< content-type: text/plain
+< vary: Accept-Encoding
+< date: Mon, 04 Nov 2024 13:21:07 GMT
+< server: Kuma Gateway
+<
+This is a Kuma MeshGateway. No routes match this MeshGateway!
+* Connection #0 to host 35.226.116.24 left intact
+```
+Notice the gateway says that there are no routes configured.
+
+## Define a route using `HTTPRoute`
+
+[HTTPRoute](https://gateway-api.sigs.k8s.io/api-types/httproute/) resources contain a set of matching criteria for HTTP 
+requests and upstream `Services` to route those requests to.
+```yaml
+echo "apiVersion: gateway.networking.k8s.io/v1
+kind: HTTPRoute
+metadata:
+  name: echo
+  namespace: kuma-demo
+spec:
+  parentRefs:
+    - group: gateway.networking.k8s.io
+      kind: Gateway
+      name: kuma
+      namespace: kuma-demo
+  rules:
+    - backendRefs:
+      - kind: Service
+        name: demo-app
+        port: 5000
+        weight: 1
+      matches:
+        - path:
+            type: PathPrefix
+            value: /" | kubectl apply -f -
+```
+
+Now try to reach our gateway again:
+```shell
+curl -v ${PROXY_IP}:8080
+```
+
+which outputs:
+```shell
+*   Trying 127.0.0.1:8080...
+* Connected to 127.0.0.1 (127.0.0.1) port 8080
+> GET / HTTP/1.1
+> Host: 127.0.0.1:8080
+> User-Agent: curl/8.4.0
+> Accept: */*
+>
+< HTTP/1.1 403 Forbidden
+< content-length: 19
+< content-type: text/plain
+< date: Fri, 09 Feb 2024 10:10:16 GMT
+< server: Kuma Gateway
+< x-envoy-upstream-service-time: 24
+<
+* Connection #0 to host 127.0.0.1 left intact
+RBAC: access denied%
+```
+
+Notice the forbidden error.
+This is because the quickstart has very restrictive permissions as defaults.
```

Comment:
Does it? Does that mean `kumactl install demo` doesn't work?

Reply:
`kumactl install demo` does work, but in quickstart we configure mTLS and create MTP that only allows traffic from demo-app to redis that is why it will be forbidden here, this is the same as in other gateway guides